src/App.tsx

```diff
@@ -15,48 +16,97 @@ export const goodsFromServer = [
   'Garlic',
 ];
 
+enum SortType {
+  NONE,
+  ALPHABET,
+  LENGTH,
+}
+
+type ReorderOptions = {
+  sortType: SortType,
+  isReversed: boolean,
+};
+
+export function getReorderedGoods(
+  goods: string[],
+  { sortType, isReversed }: ReorderOptions,
+) {
+  const visibleGoods = [...goods];
+
+  visibleGoods.sort((good1, good2) => {
+    switch (sortType) {
+      case SortType.ALPHABET: return good1.localeCompare(good2);
+
+      case SortType.LENGTH: return good1.length - good2.length;
```

Comment:
```suggestion
      case SortType.ALPHABET: 
      	return good1.localeCompare(good2);

      case SortType.LENGTH: 
      	return good1.length - good2.length;
```
It's easier to read.

Reply:
Well, maybe yore right